Gemfile.lock

```diff
@@ -573,7 +573,7 @@ DEPENDENCIES
   xmlrpc
 
 RUBY VERSION
-   ruby 3.3.0p0
+   ruby 3.3.2p78
 
 BUNDLED WITH
    2.5.18
```

Comment:
Should bundler also be updated? Latest version is 2.6.2. https://rubygems.org/gems/bundler/versions